ale_linters/javascript/flow.vim

```diff
@@ -21,11 +21,15 @@ endfunction
 
 function! ale_linters#javascript#flow#GetCommand(buffer) abort
   return ale_linters#javascript#flow#GetExecutable(a:buffer)
-  \   . ' check-contents --respect-pragma --json --from ale'
+  \   . ' check-contents --respect-pragma --json --from ale %s'
```

Comment:
This will still use stdin as input:
```
Usage: flow check-contents [OPTION]... [FILE]

Runs a flow check on the contents of stdin. If FILE is provided, then
check-contents pretends that the contents of stdin come from FILE

e.g. flow check-contents < foo.js
or   flow check-contents foo.js < foo.js
```